polaris/tasks/ocean/external_gravity_wave/__init__.py

```diff
@@ -0,0 +1,326 @@
+            ref_forward_step.set_shared_config(config, link=config_filename)
+        self.add_step(ref_forward_step, symlink=symlink)
+        analysis_dependencies['mesh'][ref_solution_factor] = base_mesh_step
+        analysis_dependencies['init'][ref_solution_factor] = init_step
+        analysis_dependencies['forward'][ref_solution_factor] = (
+            ref_forward_step
+        )
+
+        for refinement_factor in refinement_factors:
+            analysis_dependencies['mesh'][refinement_factor] = base_mesh_step
+            analysis_dependencies['init'][refinement_factor] = init_step
+
+            timestep, _ = get_timestep_for_task(
+                config, refinement_factor, refinement=refinement
+            )
+            timestep = ceil(timestep)
+            timesteps.append(timestep)
+
+            subdir = f'{case_dir}/forward/{mesh_name}_{timestep}s'
+            symlink = f'forward/{mesh_name}_{timestep}s'
+            if subdir in component.steps:
+                forward_step = component.steps[subdir]
+            else:
+                name = f'{prefix}_forward_{mesh_name}_{timestep}s'
+                forward_step = Forward(
+                    component=component,
+                    name=name,
+                    subdir=subdir,
+                    refinement_factor=refinement_factor,
+                    mesh=base_mesh_step,
+                    init=init_step,
+                    refinement=refinement,
+                    dt_type=dt_type,
+                )
+                forward_step.set_shared_config(config, link=config_filename)
+            self.add_step(forward_step, symlink=symlink)
+            analysis_dependencies['forward'][refinement_factor] = forward_step
+
+        subdir = f'{case_dir}/convergence_{refinement}/analysis'
+        if subdir in component.steps:
+            step = component.steps[subdir]
+            step.resolutions = resolutions
+            step.dependencies_dict = analysis_dependencies
+        else:
+            step = Analysis(
+                component=component,
+                subdir=subdir,
+                dependencies=analysis_dependencies,
+                refinement=refinement,
+                ref_solution_factor=ref_solution_factor,
+            )
+            step.set_shared_config(config, link=config_filename)
+        self.add_step(step)
+
+
+def _set_convergence_configs(config, prefix):
+    for refinement in ['space', 'time']:
+        option = f'refinement_factors_{refinement}'
+        refinement_factors = config.getlist(
+            'spherical_convergence', f'{prefix}_{option}', dtype=str
+        )
+        refinement_factors = ', '.join(refinement_factors)
+        config.set('convergence', option, value=refinement_factors)
+
+        base_resolution = config.getfloat(
+            'spherical_convergence', f'{prefix}_base_resolution'
+        )
+        config.set(
+            'convergence', 'base_resolution', value=f'{base_resolution:03g}'
```

Comment:
```suggestion
            'convergence', 'base_resolution', value=f'{base_resolution:g}'
```
For example, 60 km showed up as `060`